scripts/cleardown_txc.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+STAGE=$(cat .sst/stage)
+DB_CLUSTER_ARN=$(aws rds describe-db-clusters | jq -r --arg stage $STAGE '.DBClusters[] | select(.TagList[] | select((.Key=="sst:stage") and (.Value==$stage))).DBClusterArn')
+SECRET_ARN=$(aws secretsmanager list-secrets | jq -r --arg stage $STAGE '.SecretList[] | select(.Tags[]? | select(.Key=="sst:stage" and .Value==$stage))' | jq -r '. | select(.Tags[]? | select(.Key=="sst:app" and .Value=="reference-data-service"))'.ARN)
+
+declare -a arr=("service_journey_pattern_links" "service_admin_area_codes" "vehicle_journeys" "service_journey_patterns" "services")
```

Comment:
shouldn't the rest of the tables like the localities, stops, operators and services table be mentioned here?

Reply:
This is just for clearing down the txc data since it's quick to just run the retrievers for the other tables which already do this